### DISCUSSION TITLE
Welcome to governance Discussions!

### DISCUSSION THREAD

**Discussion by flyworker** [swanchain/governance#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by finalnan** [comment]
Congrats,Swan Chain LFGGGG

**Reply by Ishaqson1111** [reply]
Hello can I attend

**Comment by gugege521** [comment]
我们的社区太棒啦，我是gugege521，是swan chain的一名普通用户，从22年1月份了解到swan，并在23年7月份逐步参与到swan的激励活动中，一直都在积极为中文社区的建设贡献力量。

**Comment by hilmanraz** [comment]
wish the best for swan

**Comment by valueflowever** [comment]
LFG

**Comment by fener37** [comment]
İyi günler airdrop tarihi bellimi.

**Comment by Ataurelius** [comment]
SWAN LFG !

**Comment by retrunv0id** [comment]
when ?

**Comment by cryptoboi100k** [comment]
SWAN TYM

**Reply by Ngocda93** [reply]
Xin chúc mừng swanchain

**Comment by Taiwotimman** [comment]
Congrats on this 👏

**Comment by vincenzo2022** [comment]
congrats swan chain LFG

**Comment by Gaelion-Web3** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Congrarts Swan Chain, but why there's nothing announcement about points's snapshots
> * Share ideas.

**Reply by flyworker** [reply]
This this not a place for announcement, it is more for ideas and open discussions.

**Comment by iBlackShark42** [comment]
Nice 👍

**Comment by evengodssmileatme** [comment]
hi!
I like crypto and truly think, know it's the present and the future! I really want to understand more about blockchains, this L1 and L2, AI toolset.. etc cuz I am getting confused honestly. It's a lot of unknown words for me which I am trying to learn every day. Maybe you can help to understand about this terms and Swan more deeply in a way that even child will understand ??

**Comment by daiyiheng001** [comment]
1

**Comment by HIROZON** [comment]
Congras SwanChain

**Comment by xxScorpius97xx** [comment]
print("Hello, everyone ♨️!")

**Comment by davindersk** [comment]
Congrats Swan Chain...I m excited.
